packages/plugin-lighthouse/README.md

```diff
@@ -1,3 +1,118 @@
 # @code-pushup/lighthouse-plugin
 
-TODO: docs
+[![npm](https://img.shields.io/npm/v/%40code-pushup%2Flighthouse-plugin.svg)](https://www.npmjs.com/package/@code-pushup/lighthouse-plugin)
+[![downloads](https://img.shields.io/npm/dm/%40code-pushup%2Flighthouse-plugin)](https://npmtrends.com/@code-pushup/lighthouse-plugin)
+[![dependencies](https://img.shields.io/librariesio/release/npm/%40code-pushup/lighthouse-plugin)](https://www.npmjs.com/package/@code-pushup/lighthouse-plugin?activeTab=dependencies)
+
+🕵️ **Code PushUp plugin for measuring web performance and quality with Lighthouse.** 🔥
+
+---
+
+The plugin parses your Lighthouse configuration and lints all audits of the official [Lighthouse](https://github.com/GoogleChrome/lighthouse/blob/main/readme.md#lighthouse-------) CLI.
+
+Detected Lighthouse audits are mapped to Code PushUp audits. Audit reports are calculated based on the [original implementation](https://googlechrome.github.io/lighthouse/scorecalc/).
```

Comment:
Maybe also good to mention that Lighthouse categories are mapped to Code PushUp groups.

Reply:
Very good point!!! Love it!

Reply:
done